### ISSUE TITLE
six package is missing

### ISSUE THREAD

**Issue by cehbrecht** [conda-forge/myproxyclient-feedstock#4]
The latest PR for `myproxyclient` fails due to missing `six` package. `six` is a dependency of `pyopenssl`. Does `six` need to explicitly added to the requirements of the Python package?

**Comment by philipkershaw** [comment]
I've cut 2.1.0 to address this - hopefully fixed now

**Comment by cehbrecht** [comment]
@philipkershaw 
There is now something wrong with the license file. Error message from conda-build:
```
ValueError: License file given in about/license_file does not exist in source root dir or in recipe root dir (with meta.yaml)
```

**Comment by cehbrecht** [comment]
... `MANIFEST.in` says: 
```
include myproxy/LICENSE
```

... but it is not in the repo.

Usually the license file is in the top folder.

**Comment by philipkershaw** [comment]
Apologies, I moved it to `myproxy/client`.  This makes more sense as `myproxy` is really a namespace package.

**Comment by cehbrecht** [comment]
We need to fix it in `myproxyclient` ... conda package is generated from the pypi package.

**Comment by cehbrecht** [comment]
We still need to fix the 2.1.0 build:
https://github.com/ESGF/esgf-pyclient/issues/47

**Comment by cofinoa** [comment]
Solved at PR #8
